src/me/mrCookieSlime/Slimefun/Setup/Messages.java

```diff
@@ -72,6 +72,7 @@ public static void setup() {
 		local.setDefault("commands.research.reset-target", "&cYour Knowledge has been reset");
 		local.setDefault("machines.in-use", "&cThis Block's Inventory is currently opened by a different Player.");
 		local.setDefault("machines.ignition-chamber-no-flint", "&cIgnition Chamber is missing Flint and Steel.");
+    		local.setDefault("messages.cant-place" ,"&cYou can't place that block there!");
```

Comment:
Change "messages.cant-place" to "messages.cannot-place". This is a detail, but we have to keep a "neutral" english for convenience reasons.